src/app/home/components/SearchForm/SearchForm.tsx

```diff
@@ -0,0 +1,39 @@
+import { type ChangeEvent, type FC, type FormEvent } from 'react';
+import { SearchField } from '../SearchField/SearchField';
+import { SearchBtn } from '../SearchBtn/SearchBtn';
+import { useLocalStorage } from '../../../../hooks/useLocalStorage';
```

Comment:
use alias after fix in tsconfig
`import { useLocalStorage } from '@/hooks/useLocalStorage'`